src/main/kotlin/org/elm/openapiext/Subprocesses.kt

```diff
@@ -70,36 +71,32 @@ fun GeneralCommandLine.execute(timeoutInMilliseconds: Int? = 1000): ProcessOutpu
 
 @Throws(ExecutionException::class)
 fun GeneralCommandLine.execute(
-        owner: Disposable,
```

Comment:
Whoops, this was probably my fault for not explaining things well. The disposer code path needs to stick around. That's what `ElmCLI` calls when compiling a project (part of the CI setup and a (currently hidden) action to compile an Elm file).

Basically, there should be 2 core execution APIs:
- timeout-based, in which case no Disposer is needed (I think)
- disposer-based (can run indefinitely or until IntelliJ is closed)

The special handling for piping in stdin should be done with the timeout-based codepath since the user doesn't want to wait indefinitely (and because the code is simpler).